### PR TITLE
Add GET event/:id/builds endpoint

### PR DESCRIPTION
**Pull request by minz1027**
- Add `GET event/:id/builds` and `GET event/:id` endpoints
- So that the UI can query all builds associated with a event

### CODE REVIEW

File: plugins/events/get.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    method: 'GET',
+    path: '/events/{id}',
+    config: {
+        description: 'Get a single event',
+        notes: 'Returns a event record',
+        tags: ['api', 'events'],
+        handler: (request, reply) => {
+            const factory = request.server.app.eventFactory;
+
+            factory.get(request.params.id)
```
Comment (d2lam) [inline plugins/events/get.js]: missing `return` here

Comment (minz1027) [inline plugins/events/get.js]: fixed

Comment (petey) [inline plugins/events/get.js]: you fixed the one in builds, but not here.

File: test/plugins/jobs.test.js [hunk 2]
```diff
@@ -138,7 +138,7 @@ describe('job plugin test', () => {
         });
     });
 
-    describe('/jobs/{id}', () => {
+    describe('PUT /jobs/{id}', () => {
```
Comment (tkyi) [inline test/plugins/jobs.test.js]: nice! 👍

File: plugins/events/get.js (hunk 1 above)
Comment (minz1027) [inline plugins/events/get.js]: LOL, finally fixed!

**Pull request merged**